### ISSUE TITLE
what is the meaning of accuracy when the loss function is 'mse' such as we used in LSGAN?

### ISSUE THREAD

**Issue by szc11121** [eriklindernoren/Keras-GAN#132]
sorry for botehr but i am new to Keras. does it make a round() operation and compute the accuracy?

**Comment by eriklindernoren** [comment]
Since the target value is 0 or 1 you could clip the output of the discriminator to [0, 1] and round to compute accuracy.
